### ISSUE TITLE
bug: Icons missing in Jetbrains Console

### ISSUE THREAD

**Issue by Jelloeater** [eza-community/eza#825]
![2024-02-06_02-33](https://github.com/eza-community/eza/assets/7862369/ae1e5926-b717-4fd6-be8b-6bad35ad36f8)

If eza does something unexpected, or its output looks wrong, or it displays an error on the screen, or if it outright crashes, then please include the following information in your report:

- The version of eza being used (`eza --version`)
- v0.17.3 [+git]

- For reference
exa - list files on the command-line
v0.10.0 [+git]

- The command-line arguments you are using
eza --icons

- Your operating system and hardware platform
OS: Ubuntu MATE 22.04.3 LTS x86_64
Host: Lenovo ThinkServer TS130 110568U
Kernel: 5.15.0-53-generic
Uptime: 1 hour, 13 mins
Packages: 3306 (dpkg), 1 (brew), 7 (flatpak), 29 (snap)
Shell: zsh 5.8.1
Resolution: 1920x1200, 1920x1200, 1920x1080, 1920x1080
DE: MATE 1.26.0
WM: Metacity (Marco)
Theme: Colloid-Dark [GTK2/3]
Icons: Tela-circle-dark [GTK2/3]
Terminal: WezTerm
CPU: Intel Xeon E3-1225 V2 (4) @ 3.600GHz
GPU: NVIDIA GeForce GTX 1050 Ti
Memory: 7352MiB / 15936MiB

vs in Jetbrains
Terminal: java 

If it’s a crash, please include the full text of the crash that gets printed to the screen. If you’re seeing unexpected behaviour, a screenshot of the issue will help a lot.

See screen shot for difference.

Any ideas why? I'm using the same font in both.

In Wezterm
config.font = wezterm.font 'LigaSauceCodePro Nerd Font'

In JetBrains

![image](https://github.com/eza-community/eza/assets/7862369/87ec45ac-4d01-41fb-b98d-6150923bd6fc)

---

**Comment by cafkafk** [comment]
This might be an outdated version of nerdfonts, what version are you running on.

**Comment by Jelloeater** [comment]
I've been using this one... which is a little old.
https://github.com/Bo-Fone/Liga-Sauce-Code-Pro-Nerd-Font
It's a further patched one w/ ligatures.

**Comment by Jelloeater** [comment]
![image](https://github.com/eza-community/eza/assets/7862369/947a33df-b59e-44e9-bb38-21402b0e0385)
Also seems to work fine on my work Mac
Pycharm CE on the left, iTerm2 on the right
![image](https://github.com/eza-community/eza/assets/7862369/6be5de87-4410-4efe-9d0b-9cc71b777567)
![image](https://github.com/eza-community/eza/assets/7862369/46753009-28c8-4170-b881-0721490ad942)

**Comment by Jelloeater** [comment]
I have a hunch that it's due to exa being quite old AND the fonts being about as old, so they work with each other.

**Comment by Jelloeater** [comment]
Oh man, my iTerm2 was using the newer version, which is why it looked better.
Sauce Code Pro Nerd Font Complete seems to work better, but still getting some spacing issues.

**Comment by cafkafk** [comment]
Do you have any spacing issues without missing icons? We've seen those with some unicode symbols in names and such sometimes (screenshots?)?

**Comment by Jelloeater** [comment]
Swapping encoding didn't help w/ spacing on JetBrains

**Comment by Jelloeater** [comment]
Also seeing this as well
![image](https://github.com/eza-community/eza/assets/7862369/b9f6b03c-dacd-4d5d-b9f0-c6ed07e05681)
vs
![image](https://github.com/eza-community/eza/assets/7862369/079f73fb-c57a-4409-b862-bde5c5881987)

with exa --long --extended --links --header --git --icons --classify -a -a
vs 
eza --long --extended --links --header --git --icons --classify -a -a

**Comment by binury** [comment]
So, @Jelloeater you don't see the issue anymore in iTerm after swapping out the font?

Just confirming the initial issue: it does seem like there is some issue related to [some] icons and terminal compatibility, irrespective of NerdFont version(s)… Potentially unrelated to eza but to some specific settings?

<img width="898" alt="ss 2024-04-15 at 09 40 51@2x" src="https://github.com/eza-community/eza/assets/1146921/69d1523c-9cf5-415f-85e4-32b4d8a12d8a">


<img width="906" alt="ss 2024-04-15 at 09 40 26@2x" src="https://github.com/eza-community/eza/assets/1146921/39162398-4d57-4091-9c00-14dd84536347">


I just picked a new NerdFont to download and test, and observe the glpyhs missing in some terminals all using the same font family

**Comment by binury** [comment]
Well, it seems like it is a terminal issue in case of iTerm, potentially.

https://gitlab.com/gnachman/iterm2/-/issues/10133


This is thankfully resolved in the [latest versions](https://iterm2.com/downloads/nightly/#/section/home)– but still not yet in the 'stable' releases, despite having been fixed over a year ago 🤷

**Comment by wytsai7660** [comment]
> Well, it seems like it is a terminal issue in case of iTerm, potentially.
> 
> https://gitlab.com/gnachman/iterm2/-/issues/10133
> 
> This is thankfully resolved in the [latest versions](https://iterm2.com/downloads/nightly/#/section/home)– but still not yet in the 'stable' releases, despite having been fixed over a year ago 🤷

Yes! I once encountered the same issue and I also posted a discussion. https://github.com/orgs/eza-community/discussions/691#discussion-5899192.

After switching to beta release, this issue is fixed.
<img width="922" alt="Screenshot 2024-04-16 at 00 03 05" src="https://github.com/eza-community/eza/assets/116703995/fddcae0e-5eac-4630-95ec-4095a706f9c1">
